spec/controllers/holds_controller_spec.rb

```diff
@@ -189,10 +189,30 @@
     end
 
     describe '#result' do
-      it 'renders the result template' do
-        get :result
+      context 'when not redirected from /holds/new' do
+        it 'redirects to the summary page' do
+          get :result
 
-        expect(response).to render_template(:result)
+          expect(response).to redirect_to summaries_path
+        end
+      end
+
+      context 'when redirected from /holds/new' do
```

Comment:
this might be redundant because we are testing the same thing for create
since this is a controller spec, cannot use sth like follow_redirect! to test the rendered template
that can be done with a request spec though, not sure if we need that.

let me know your thoughts..

Reply:
Yeah I would get rid of this second spec because it's redundant as you say.

Reply:
ok, will do, wanted to see what you think 👍